server/internal/usecase/interactor/group.go

```diff
@@ -242,3 +243,28 @@ func (i Group) getModelsByGroup(ctx context.Context, g *group.Group) (res model.
 	}
 	return
 }
+
+func (i Group) UpdateOrder(ctx context.Context, ids id.GroupIDList, operator *usecase.Operator) (group.List, error) {
```

Comment:
What if the user sends a list of group IDs from multiple projects? or what if the user sends a subset of the project's groups?

I guess this implementation needs to be enhanced